partcad-cli/src/partcad_cli/click/commands/lint.py

```
@@ -0,0 +1,76 @@
+async def cli_lint_async(ctx: Context, packages: list[str], filter_prefix: str) -> None:
+    tasks = []
+    lint_checks = get_linting_checks(user_config.threads_max)
+    if filter_prefix:
+        lint_checks = list(filter(lambda l: l.name.startswith(filter_prefix), lint_checks))
+        pc.logging.debug(f"Running lint checks with prefix {filter_prefix}")
+
+    for package in packages:
+        prj = ctx.get_project(package)
+        tasks.extend(
+          [
+            l.lint_log_wrapper(ctx, prj, t) for l in lint_checks \
```

Ruff (0.11.9)

Ambiguous variable name: `l`

(E741)